ai_eval/templates/coding_ai_eval.html

```diff
@@ -0,0 +1,33 @@
+<div>
+  <div id="question-text"></div>
+</div>
+<div class="used-prog-language">
+  {{ self.language }}
+</div>
```

Comment:
It can't be a `<select> `element because only the instructor is supposed to have the ability to change it. I liked the design, so I styled it as a select-like badge to maintain the original mock-up's look and feel

Reply:
@CefBoud I see. 

> because only the instructor is supposed to have the ability to change it

This means, the mockups are outdated? Can we ask for the mockups to reflect this, so that we have a better idea of how to style this element?